Automattic/woocommerce-payments
# Disputes: Add dispute notice to transactions page

Fixes #6923 

#### Changes proposed in this Pull Request

This PR adds a dispute warning notice is added to the transaction page. This warning is part of #6947 and is hidden behind a feature flag.

The notice is shown as yellow if the dispute is awaiting a response and is not yet due. The notice changes to red when there are less than 3 days left to respond.

![Screenshot 2023-08-15 at 4 22 23 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/5f76e592-e9a3-4523-8f6f-6317129ce5a8)
![Screenshot 2023-08-15 at 4 21 19 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/f94b2990-ddde-49fc-b931-34edbf82a65e)

**Inquiry:**
![image](https://github.com/Automattic/woocommerce-payments/assets/57298/fd8c32b4-cafb-46d0-aead-aa3e3979176e)

> The cardholder claims this is an unauthorized transaction. Challenge the inquiry if you believe the claim is invalid. Non-response will result in an automatic loss. Learn more about responding to disputes and inquiries






#### Testing instructions

1. Enable feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/) or by modifying your database table `wp_options` directly.
2. Load a Transaction Details page of an active dispute (`/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Ftransactions%2Fdetails&id=pi_abc&transaction_id=txn_abc&transaction_type=charge`).
3. See the Dispute Notice. 

-------------------


**Post merge**
